### PR TITLE
Update to reflect CascadiaCSS and some updates.

### PR DESCRIPTION
**Pull request by crtr0**
Let me know what you think!

### CODE REVIEW

File: proposals/README.md [hunk 1]
```diff
@@ -1,54 +1,65 @@
-## CASCADIAJSFEST CALL FOR PROPOSALS – THE DETAILS
+# CASCADIA FEST CALL FOR PROPOSALS – THE DETAILS
 
-CJSFest 2015 is July 8, 9, and 10th near Seattle, Washington, and we'd love for you to be a part of it!
-Each day is organized around a theme. Share your expertise with the Cascadia community on:  
-* CSS Day
-* BrowserJS Day
-* ServerJS Day
+Cascadia Fest 2015 is taking place July 8-10 in northen Washington State and we'd love for you to be a part of it! This is our first year running a Fest and it is made up of two events over 3 days:
```
Comment (alexlande) [inline proposals/README.md]: Cascadia Fest is a cool name :+1:

File: proposals/README.md [hunk 2]
```diff
@@ -1,54 +1,65 @@
...
 discussions. You can indicate your format in the CfP submission.
 
-### HOW TO SUBMIT A PROPOSAL
+### CascadiaCSS
+
+CascadiaCSS is brand-new this year. We will spend a day exploring the latest and greatest technologies and techniques related to building amazing user experiences for the web, very much in the spirit of [CSSConf](http://2015.cssconf.com/). Some topics we'd like to explore:
+
+- User experience design
+- Advanced CSS
+- SCSS / SASS / LESS
```
Comment (alexlande) [inline proposals/README.md]: Replace with `Sass / LESS / Stylus / PostCSS` (The Sass community gets very sensitive about only referring to it as "Sass")

File: proposals/README.md [hunk 3]
```diff
@@ -1,54 +1,65 @@
...
+- Responsive design
+- Cross-platform tips & tricks
+- SVG
+- Tooling
```
Comment (alexlande) [inline proposals/README.md]: Can we add Accessibility to this list?

**Comment by alexlande** [comment]
A few minor nits, but looks great!

File: proposals/README.md (hunk 1 above)
Comment (hackygolucky) [inline proposals/README.md]: I think 'first year running a Fest' might confuse people who don't know we've existed prior to this year.

File: proposals/README.md [hunk 4]
```diff
@@ -1,54 +1,65 @@
...
+pitches (i.e. How to Use Product X). A great talk weaves a story -- it can address issues that
 developers struggle with, break down and explain complicated topics, and cover
 new ground in JS technology.  All the talks will be 30 minutes. We don't offer
 Q&A following a talk. We are framework agnostic. We also accept non-talk proposals such as moderated
 discussions. You can indicate your format in the CfP submission.
 
-### HOW TO SUBMIT A PROPOSAL
+### CascadiaCSS
+
+CascadiaCSS is brand-new this year. We will spend a day exploring the latest and greatest technologies and techniques related to building amazing user experiences for the web, very much in the spirit of [CSSConf](http://2015.cssconf.com/). Some topics we'd like to explore:
```
Comment (hackygolucky) [inline proposals/README.md]: suggestion: "CascadiaCSS is an exciting addition to Cascadia this year..."

**Comment by crtr0** [comment]
Pushed a commit with edits. Feel free to merge if it looks good!

**Comment by alexlande** [comment]
:+1: :octopus: :rocket:

**Comment by hackygolucky** [comment]
:koala: :cake:

**Pull request merged**